new.php

```diff
@@ -230,13 +230,12 @@ function set_progress( float $pc, string $label ) {
 	$patchesApplied[] = $data[0]['_number'] . ',' . $data[0]['revisions'][$revision]['_number'];
 	$commands[] = [
 		[
-			'PATCHDEMO' => __DIR__ . '/',
```

Comment:
The script uses this variable.

Reply:
Bit of an unrelated change, but PATCHDEMO is part of $baseEnv so doesn't need to be repeated here